src/EditorFeatures/Core/IntelliSense/AsyncCompletion/CompletionSource.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Was this change intentional? It appears the file in the repository already has the correct encoding.

Reply:
fixed